<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QBalancingWidget</class>
 <widget class="QWidget" name="QBalancingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <widget class="QFrame" name="controls_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="idle_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>idle</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/balancing_control_idle.png</normaloff>:/icons/balancing_control_idle.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="attitude_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>attitude</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/balancing_control_attitude.png</normaloff>:/icons/balancing_control_attitude.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="velocity_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>velocity</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/balancing_control_velocity.png</normaloff>:/icons/balancing_control_velocity.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="position_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>position</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/balancing_control_position.png</normaloff>:/icons/balancing_control_position.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="widgets_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPolarPlot" name="balance_widget" native="true"/>
      </item>
      <item>
       <widget class="QMapWidget" name="map_widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPolarPlot</class>
   <extends>QWidget</extends>
   <header>qpolarplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QMapWidget</class>
   <extends>QWidget</extends>
   <header>qmapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
